components/feed/feed.jsx

```diff
@@ -43,29 +43,37 @@ export default function Feed() {
   return (
     <>
     <div className={styles.cardContainer}>
-    <Carousel verticalMode itemsToShow={1} showArrows={false} onChange={(currentItem, pageIndex) =>
+    <Carousel verticalMode itemsToShow={1} showArrows={false} pagination={false} onChange={(currentItem, pageIndex) =>
     handleSwipe(pageIndex)}>
       {posts.map((post,index) =>
         <div className={styles.card} key={index}>
           <div className={styles.feedContainer}>
+            <span className={styles.home}>HOME</span>
             <span className={styles.username}>{post.username}</span>
             <span className={styles.date}>{(new Date(post.date)).toLocaleDateString('en-US')}</span>
-            <span className={styles.category}>{post.category}</span>
           </div>
+
           <div className={styles.post}>
-            <p className={styles.topic}>{post.title}</p>
+            <div className={styles.topic}>
+            <div className={styles.category}>{post.category}</div>
+             {post.title}
+            </div>
             <p className={styles.text}>{post.body}</p>
             <div className={styles.logo}>
             <Image src={logo} alt='logo'/>
             <span className={styles.interactions}>{post.interactions}</span>
             </div>
+           <div className={styles.effect}>
+            {votes < -99 ? <Lottie lotti={animationDislike} height={300} width={300} speed={0.5}/> : null}
+            {votes > 99 ? <Lottie lotti={animationConfetti} height={500} width={500} speed={0.5}/> : null}
+           </div>
           </div>
-          <div className={styles.field}>
-            <div className={styles.sliderLeft}>-100</div>
+
+          {email !== null ? <div className={styles.field}>
             <input className={styles.slider} type='range' min='-100' max='100' value={votes} steps='1'
-            onChange={(e)=>{setVotes(e.target.value);}}></input>
-            <div className={styles.sliderRight}>100</div>
-          </div>
+            onChange={(e)=>{setVotes(e.target.value); }}></input>
```

Comment:
The setVotes needs to reset the slider value upon new card. Currently if slider is moved on a card, it will persist through following cards.